lib/bolt/config.rb

```diff
@@ -127,19 +137,20 @@ def update_from_file(data)
 
       if data['pcp']
         if data['pcp']['service-url']
-          self[:transports][:pcp][:service_url] = data['pcp']['service-url']
+          self[:transports][:pcp][:"service-url"] = data['pcp']['service-url']
         end
         if data['pcp']['cacert']
           self[:transports][:pcp][:cacert] = data['pcp']['cacert']
         end
         if data['pcp']['token-file']
-          self[:transports][:pcp][:token_file] = data['pcp']['token-file']
+          self[:transports][:pcp][:"token-file"] = data['pcp']['token-file']
         end
         if data['pcp']['task-environment']
-          self[:transports][:pcp][:orch_task_environment] = data['pcp']['task-environment']
```

Comment:
oh hmm I wasn't properly testing task environment.

This doesn't get '-' because it's not passed on the client?

Reply:
I could make it consistent with the other two. `token-file` and `service-urls` aren't accessible on the CLI either.

It probably doesn't get a dash because it's not available in the Orchestrator client config.

Reply:
"It probably doesn't get a dash because it's not available in the Orchestrator client config."

thats what I was getting at. i was more curious about the reasoning.